FEXCore/Scripts/config_generator.py

```diff
@@ -468,6 +468,10 @@ def print_parse_jsonloader_options(options):
                 output_argloader.write("else if (KeyName == \"{0}\") {{\n".format(op_key))
                 output_argloader.write("Set(KeyOption, FEXCore::Config::EnumParser<FEXCore::Config::{}ConfigPair>(FEXCore::Config::{}_EnumPairs, Value_View));\n".format(op_key, op_key, op_key))
                 output_argloader.write("}\n")
+            elif (value_type == "strarray"):
+                output_argloader.write("else if (KeyName == \"{0}\") {{\n".format(op_key))
+                output_argloader.write("AppendStrArrayValue(KeyOption, ConfigString);\n")
```

Comment:
should this have a `\t` before the append to indent it? It's generated code so it doesn't matter either way, but

Reply:
Sure, I added a \t there to match some of the other generated behaviour.